hi-ml-multimodal/notebooks/phrase_grounding.ipynb

```diff
@@ -180,6 +180,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "text_prompt = \"cardiomegaly\"\n",
```

Comment:
Could we change this to a phrase, for example "There is cardiomegaly"? We expect the model to do better when we use phrases instead of single words. Since this is a demo notebook, it would be nice to make the examples as realistic as possible.

Reply:
Sounds good to me! Feel free to commit or suggest.

Reply:
I've suggested it.